projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.php

```diff
@@ -67,6 +82,20 @@ function wpcom_enqueue_admin_bar_assets() {
 CSS
 		);
 	}
+
+	$admin_color      = get_user_option( 'admin_color' );
+	$admin_icon_color = WPCOM_ADMIN_ICON_COLORS[ $admin_color ] ?? WPCOM_ADMIN_ICON_COLORS['fresh'];
+
+	// Force the icon colors to have desktop color even on mobile viewport.
+	wp_add_inline_style(
+		'wpcom-admin-bar',
+		<<<CSS
+			#wpadminbar.mobile .quicklinks .ab-icon:before,
+			#wpadminbar.mobile .quicklinks .ab-item:before {
+				color: $admin_icon_color !important;
+			}
```

Comment:
This only works for left-side icons and not right-side ones.

@DustyReagan, I saw you were going to update the help center, notification icons etc as part of https://github.com/Automattic/dotcom-forge/issues/8454. Can we try updating the icons so that they follow how the left-side icons are declared? E.g. this is how the wpcom logo SVB is declared in the CSS:

https://github.com/Automattic/jetpack/blob/5aef951f8fe4f784c44a951a66073b55d22443ff/projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss#L31-L33

If we do that, the new icons will automatically follow the Desktop color for free. Please consider 🙏